src/newrelic_airflow_plugin/newrelic_plugin.py

```diff
@@ -59,7 +59,8 @@ def harvester(cls):
                 return harvester
 
             insert_key = os.environ["NEW_RELIC_INSERT_KEY"]
-            client = MetricClient(insert_key)
+            host = os.environ.get("NEW_RELIC_HOST", "metric-api.newrelic.com")
+            client = MetricClient(insert_key, host)
```

Comment:
```suggestion
            client = MetricClient(insert_key, host=host)
```

For the sake of safety I would prefer passing host as a keyword arg. Just bikeshedding though.